content/post/personas.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Who is the typical user of SONAR?"
+date: 2019-05-05T11:33:58+02:00
+draft: false
+tags: ["personas", "user interface", "user-centered design"]
+---
+
+An important task of the SONAR project is to conceive a user-friendly interface. We decided to structure this process according to the [five elements](http://www.jjg.net/elements/pdf/elements_ch02.pdf) of J.J. Garrett[^1]. Firstly, define the strategy of your product, and then its scope, its structure, its skeleton and finally its surface.
+
+<!--more-->
+
+Defining the typical user of SONAR is part of the element *scope* (user needs). To this aim, we adopted a method based on personas.
+
+### What is a persona?
+
+The *persona* is a well-known concept in the interaction design community. It was introduced 1999 by Alan Cooper[^2], who defines it as follows:
+
+> We make up pretend users and design for them. We call these pretend users personas and they are the necessary foundation of good interaction design. Personas are not real people, but they represent them throughout the design process. They are hypothetical archetypes of actual users. Although they are imaginary, they are defined with significant rigor and  precision. Actually, we don't so much "make up" our personas as discover them as a byproduct of the investigation process. We do, however, make up their names and personal details.
+
+### The 3 main SONAR personas
+
+Each persona is presented below with an image, a name, some minimal context information and the tasks he/she performs on SONAR. The tasks are listed for both services: the shared and the dedicated portal (as described in the [Service](/service) section).
+
+|   |   |
+| ------ | ----------- |
+| <img class="image" src="/images/persona1.svg" alt="Population persona" title="Population persona" ><br/>**Fabrice**<br/>*Student (University of Kinshasa)* | **Shared or dedicated portal**<ul><li>access full text of publications for free so that I can keep up to date with scientific news</li><li>access publication metadata so that I can include references in my own publications</li><li>search publications using various criteria (author, project, institution, document type, etc.)</li></ul> |
```

Comment:
`| <img class="image" src="/images/persona1.svg" alt="Population persona" title="Population persona"  style="vertical-align:top;"><br/>**Fabrice**<br/>*Student (University of Kinshasa)* |`

`style="vertical-align:top;"` does the trick. To be added to every persona picture of the table.